src/shivangi/Test1_NonRepeatedCharInString.java

```diff
@@ -0,0 +1,30 @@
+package shivangi;
+
+public class Test1_NonRepeatedCharInString {
+	
+	int findRepeatingCharCount(String name,char ch) {
+		int count=0;
+		for(int index=0; index<name.length(); index++) {
+			if(name.charAt(index)==ch) {
+				count++;
+			}
+		}
+		return count;
+	}
+
+	void findNonRepeatCharInString(String name) {
+		for(int index=0;index<name.length();index++) {
+			int count = findRepeatingCharCount(name,name.charAt(index));
+			if(count==1) {
+				System.out.println("First non repeating charcater ---- "+ name.charAt(index));
```

Comment:
here you can use 
indexOf(ch) ==lastIndexOf(ch) logic to avoid one more for loop

Reply:
Okay @KrishnaKTechnocredits mam will try using this statement of code.